### PR TITLE
PICARD-1011: Update tab order to match Action Toolbar order

### PR DESCRIPTION
**Pull request by Sophist-UK**
The order of icons on the Action Toolbar was changed in #564.

This PR changes the tab order to match.

Resolves https://tickets.metabrainz.org/browse/PICARD-1011

### CODE REVIEW

**Comment by Sophist-UK** [comment]
P.S. I still need to test this patch.

File: picard/ui/mainwindow.py [hunk 1]
```diff
@@ -654,17 +654,17 @@ def set_tab_order(self):
...
-        tab_order(tw(self.play_file_action), tw(self.save_action))
-        tab_order(tw(self.save_action), tw(self.submit_acoustid_action))
-        tab_order(tw(self.submit_acoustid_action), tw(self.cd_lookup_action))
-        tab_order(tw(self.cd_lookup_action), tw(self.cluster_action))
+        tab_order(tw(self.add_files_action), tw(self.cluster_action))
         tab_order(tw(self.cluster_action), tw(self.autotag_action))
         tab_order(tw(self.autotag_action), tw(self.analyze_action))
-        tab_order(tw(self.analyze_action), tw(self.view_info_action))
+        tab_order(tw(self.analyze_action), tw(self.browser_lookup_action))
+        tab_order(tw(self.browser_lookup_action), self.save_action)
```
Comment (samj1912) [inline picard/ui/mainwindow.py]: Picard crashes on start.
change this to 

```python
tab_order(tw(self.browser_lookup_action), tw(self.save_action))
```

**Comment by Sophist-UK** [comment]
Hmm - I have just noticed the new Customise Action Toolbar functionality.

I suspect that this code needs to be junked and tab order functionality added to the Customise Action Toolbar functionality. What do others think?

**Comment by samj1912** [comment]
Yes. Made an alternative patch with dynamic update to the tab order. See #655

**Pull request closed without merging**